### PR TITLE
Sample to test sonarqube github plugin

### PR DESCRIPTION
**Pull request by mhpnguyen**
Please don't merge. This is test only

### CODE REVIEW

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public TrustMode() {
     public PRPAIN201306UV02 processResponse(ResponseParams params) {
         LOG.debug("Begin TrustMode.processResponse()...");
         PRPAIN201306UV02 response = null;
-
+        int sonargithub = 1;
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java]: MAJOR Remove this useless assignment to local variable "sonargithub". [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1854)
MAJOR Remove this unused "sonargithub" local variable. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1481)

**Comment by mhpnguyen** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by mhpnguyen** [comment]
retest this please

**Comment by mhpnguyen** [comment]
retest this please

**Comment by mhpnguyen** [comment]
retest this please

**Comment by mhpnguyen** [comment]
retest this please

**Comment by mhpnguyen** [comment]
retest this please

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 2]
```diff
@@ -66,7 +66,10 @@ public TrustMode() {
...
+        if (sonargithub ==2){
+            System.out.println("Please don't merge this");
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS106)

**Comment by mhpnguyen** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by mhpnguyen** [comment]
Please test this

**Comment by mhpnguyen** [comment]
test this please

**Comment by mhpnguyen** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by mhpnguyen** [comment]
test this please

**Comment by mhpnguyen** [comment]
test this please

**Comment by mhpnguyen** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 3]
```diff
@@ -66,7 +66,13 @@ public TrustMode() {
     public PRPAIN201306UV02 processResponse(ResponseParams params) {
         LOG.debug("Begin TrustMode.processResponse()...");
         PRPAIN201306UV02 response = null;
-
+        int sonargithub = 1;
+        //TODO: remove this
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:70]: INFO Complete the task associated to this TODO comment. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1135)

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 4]
```diff
@@ -66,7 +66,13 @@ public TrustMode() {
...
+        if (sonargithub ==2){
+            String removeLine = "remove me";
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:72]: MAJOR Remove this useless assignment to local variable "removeLine". [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1854)
MAJOR Remove this unused "removeLine" local variable. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1481)

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 5]
```diff
@@ -66,7 +66,13 @@ public TrustMode() {
...
+            //System.out.println("Please don't merge this");
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:73]: MAJOR This block of commented-out lines of code should be removed. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java [hunk 6]
```diff
@@ -66,7 +66,13 @@ public TrustMode() {
...
+            System.out.println("Please don't merge it again");
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:74]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS106)

**Comment by mhpnguyen** [comment]
SonarQube analysis reported 5 issues:
- MAJOR 4 major
- INFO 1 info

Watch the comments in this conversation to review them.

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java (hunk 4 above)
Comment (christophermay07) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:72]: This is pretty freaking awesome!  Will the plugin tackle each commit one by one, or just the entire PR at once?

Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java:72]: The plugin will monitor and tackle the entire PR at once.  We hope to reduce sonarqube bug as we do PR.

**Pull request closed without merging**